### PR TITLE
Switch to flit build backend

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,14 +1,79 @@
...
+Source = "https://github.com/jupyterlab/jupyterlab_server"
+Tracker = "https://github.com/jupyterlab/jupyterlab_server/issues"
+
+[project.optional-dependencies]
+openapi = [
+  "openapi_core>=0.14.2",
+  "ruamel.yaml",
+]
+test = [
+  "openapi_core>=0.14.2",
```
Comment (bollwyvl) [inline pyproject.toml:56]: might as well sort

File: pyproject.toml [hunk 2]
```diff
@@ -1,14 +1,79 @@
...
+  "jupyter_server[test]",
+  "openapi-spec-validator<0.5",
+  "pytest-console-scripts",
+  "pytest-cov",
+  "pytest>=5.3.2",
+  "strict-rfc3339"
+]
+
+[project.license]
+file = 'LICENSE'
```
Comment (bollwyvl) [inline pyproject.toml:69]: maybe normalize quotes

File: pyproject.toml (hunk 1 above)
Comment (blink1073) [reply pyproject.toml]: Done!

File: pyproject.toml (hunk 2 above)
Comment (blink1073) [reply pyproject.toml]: Done!

**Comment by bollwyvl** [comment]
it's so pretty i love it

**Comment by blink1073** [comment]
> it's so pretty i love it

Yeah it is a yummy diff

**Pull request merged**